electrumsv/network_support/esv_client.py

```diff
@@ -61,6 +65,13 @@
                        "CbYXKGNm4V_2HKr68ES5bytZ8F19IS0XbJlq41accQ=="
 
 
+def chain_tip_to_header_obj(tip: bytes) -> Header:
```

Comment:
I have no idea what this is doing. What is a chain tip? Up til now a tip is just the header at the end of a chain. This uses it for something differently.

Reply:
Additionally it should be using the bitcoinx `Network.deserialized_header` function.